rails_app/app/adapters/graphite.rb

```diff
@@ -0,0 +1,60 @@
+require 'net/http'
+require 'json'
+require 'graphite-api'
+
+class GraphiteAdapter
+    def initialize
```

Comment:
yo use two spaces for indentation

Reply:
It makes me feel like javascript, ugh!
done